src/controller/CommissioningDelegate.h

```diff
@@ -679,6 +679,21 @@ struct GeneralCommissioningInfo
     ;
 };
 
+// ICDManagementClusterInfo is updated when controller read ICDManagementCluster, and used to instruct whether to go with ICD Configuration
+// and further indicate to the user how to trigger the ICD to switch to active mode.
+struct ICDManagementClusterInfo
+{
+    // Depict whether it is LIT or SIT device
+    bool isLIT;
+    bool checkInProtocolSupport;
+
+    // UserActiveModeTriggerHint indicates which user action(s) will trigger the ICD to switch to Active mode.
+    // For a LIT:  UserActiveModeTriggerHint is mandatory, For a SIT: UserActiveModeTriggerHint may not be there.
+    // UserActiveModeTriggerInstruction is dependent upon theref_UserActiveModeTriggerHint>> and may not be there.
+    BitMask<app::Clusters::IcdManagement::UserActiveModeTriggerBitmap> userActiveModeTriggerHint;
+    CharSpan userActiveModeTriggerInstruction;
```

Comment:
```suggestion
    // userActiveModeTriggerHint indicates which user action(s) will trigger the ICD to switch to Active mode.
    // For a LIT: The device is required to provide a value for the bitmap.  
    // For a SIT: The device may not provide a value.  In that case, none of the bits will be set.
    //
    // userActiveModeTriggerInstruction may provide additional information for users for some specific
    // userActiveModeTriggerHint values.
    BitMask<app::Clusters::IcdManagement::UserActiveModeTriggerBitmap> userActiveModeTriggerHint;
    CharSpan userActiveModeTriggerInstruction;
```

Looks like a bad copy-paste from the spec, with that `ref_` bit?